src/internal/testutil/auth.go

```diff
@@ -47,6 +47,12 @@ func ActivateAuth(tb testing.TB) {
 	activateAuthHelper(tb, client)
 }
 
+// ActivateAuthClient activates the auth service in the test cluster, if it isn't already enabled
+func ActivateAuthClient(tb testing.TB, c *client.APIClient) {
```

Comment:
Can you clarify why we need to maintain the old `ActivateAuth`?

Reply:
`ActivateAuth` can go away once the migration towards this model is complete. There are still several cmd tests using the current approach that would need to be migrated first